### ISSUE TITLE
[New Rule] Merge duplicate startswith / endswith calls

### ISSUE THREAD

**Issue by Skylion007** [MartinThoma/flake8-simplify#169]
## Explanation

Explain briefly why you think this makes the code simpler.
This is similar to the isinstance call merging. You can also merge startswith, endswith calls on strings. This means the builtin method only needs to iterate through the string once. [flake8-PIE](https://github.com/sbdchd/flake8-pie) already enables this rule as PIE810.

## Example

```python
# Bad
a = "abcdef"
if a.startswith("abc") or a.startswith("xyz"):
     print("alphabet found")

# Good
a = "abcdef"
if a.startswith("abc", "xyz"):
     print("alphabet found")
```
can be repeated with endswith as well

**Comment by janosh** [comment]
Came here to suggest the same rule. 👍 

Note that the good example is missing parens:

```diff
- if a.startswith("abc", "xyz"):
+ if a.startswith(("abc", "xyz")):
```
